waspc/src/Wasp/Generator/ServerGenerator/ApiRoutesG.hs

```diff
@@ -0,0 +1,68 @@
+module Wasp.Generator.ServerGenerator.ApiRoutesG
+  ( genApis,
+  )
+where
+
+import Data.Aeson (object, (.=))
+import qualified Data.Aeson as Aeson
+import Data.Char (toLower)
+import StrongPath (Dir, File', Path, Path', Posix, Rel, reldirP, relfile)
+import qualified StrongPath as SP
+import Wasp.AppSpec (AppSpec)
+import qualified Wasp.AppSpec as AS
+import Wasp.AppSpec.Api (Api)
+import qualified Wasp.AppSpec.Api as Api
+import Wasp.Generator.Common (ServerRootDir, makeJsonWithEntityData)
+import Wasp.Generator.FileDraft (FileDraft)
+import Wasp.Generator.Monad (Generator)
+import qualified Wasp.Generator.ServerGenerator.Common as C
+import Wasp.Generator.ServerGenerator.JsImport (getJsImportStmtAndIdentifier)
+
+genApis :: AppSpec -> Generator [FileDraft]
+genApis spec =
+  sequence
+    [ genApiRoutes spec,
+      genApiTypes spec
+    ]
+
+genApiRoutes :: AppSpec -> Generator FileDraft
+genApiRoutes spec =
+  return $ C.mkTmplFdWithDstAndData tmplFile dstFile (Just tmplData)
+  where
+    apis = map snd $ AS.getApis spec
+    tmplData = object ["apiRoutes" .= map getTmplData apis]
+    tmplFile = C.asTmplFile [relfile|src/routes/apis/index.js|]
+    dstFile = SP.castRel tmplFile :: Path' (Rel ServerRootDir) File'
+
+    getTmplData :: Api -> Aeson.Value
+    getTmplData api =
+      let (jsImportStmt, jsImportIdentifier) = getJsImportStmtAndIdentifier relPathFromApisDirToServerSrcDir (Api.fn api)
```

Comment:
Too bad this is not yet merged, but we could refactor this in the future to use `extImportToImportJson` https://github.com/wasp-lang/wasp/pull/998#diff-c45766197d8c820bbae67e251a4a002a1845a2da09f9d17a89c6819919241940R18